Timer/force-app/main/default/lwc/timerLWC/timerLWC.html

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div
+    style="
+      background: white;
+      height: 450px;
+      width: 600px;
+      padding: 40px;
+      border-radius: 10px;
+    "
+  >
+    <div style="font-size: 40px; text-align: center; height: 100px">
+      <template if:true={showHours}> {hours}h : </template>
+      <template if:true={showMinutes}> {minutes}m : </template>
+      <template if:true={showSeconds}> {seconds}s </template>
+    </div>
+
+    <div style="height: 100px">
+      <template if:true={showHoursMinutesSeconds}>
+        <lightning-layout vertical-align="center">
+          <lightning-layout-item size="1.5">
+            <lightning-input
+              name="hours"
+              type="number"
+              style="margin-left: 10px"
+              label="Hours"
+              placeholder="00h"
+              onchange={handleChange}
+            >
+            </lightning-input>
+          </lightning-layout-item>
+          <lightning-layout-item size="1.5">
+            <lightning-input
+              name="minutes"
+              type="number"
+              style="margin-left: 15px"
+              label="Minutes"
+              placeholder="00m"
+              onchange={handleChange}
+            >
+            </lightning-input>
+          </lightning-layout-item>
+
+          <lightning-layout-item size="1.5">
+            <lightning-input
+              name="seconds"
+              type="number"
+              style="margin-left: 15px"
+              label="Seconds  "
+              placeholder="00s"
+              onchange={handleChange}
+            >
+            </lightning-input>
+          </lightning-layout-item>
+        </lightning-layout>
+      </template>
+    </div>
+
+    <div style="height: 50px">
```

Comment:
Why are we not using flex for writing CSS?
We had a purpose behind learning it.